src/App.js

```diff
@@ -1,26 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <Calculator />
```

Comment:
- [ ] As mentioned by the previous reviewer, Let's make the App.js component class-based. [Read more about class-based components here](https://www.w3schools.com/react/react_class.asp).